README.md

```diff
@@ -67,13 +69,17 @@ An example cross platform middle tier api is provided as part of the solution, i
     - Install kubectl on Windows with PowerShell
         ```powershell
         Install-Script -Name install-kubectl -Scope CurrentUser -Force
-        install-kubectl.ps1 [-DownloadLocation <path>]
+        install-kubectl.ps1 -DownloadLocation <path>
+            - where <path> is the location you wish to install it. 
+            - you may be prompted to install a new version of nuget.  do it. 
         ```
 6. ACS Engine
     - [Download and install ACS Engine](https://github.com/Azure/acs-engine/blob/master/docs/acsengine.md#install)
+        - be sure and put (or add) acs-engine.exe to your $PATH
```

Comment:
👍

Reply:
The first two lines on the page the url take you to are (emphasis mine):
>Binary downloads for the latest version of acs-engine for are available here. Download acs-engine for your operating system. **Extract the binary and copy it to your $PATH**.

I don't see why we need to repeat the instructions here.